### PR TITLE
Add Ruby versions 2.6.10, 2.7.6, 3.0.4

### PR DESCRIPTION
**Pull request by ain**
- https://www.ruby-lang.org/en/news/2022/04/12/ruby-2-6-10-released/
- https://www.ruby-lang.org/en/news/2022/04/12/ruby-2-7-6-released/
- https://www.ruby-lang.org/en/news/2022/04/12/ruby-3-0-4-released/

### CODE REVIEW

File: ruby-pipeline/ruby-latest.yaml [hunk 1]
```diff
@@ -62,17 +62,20 @@ steps:
            '--prebuilt-image=2.6.7=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.6.7:latest',
            '--prebuilt-image=2.6.8=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.6.8:latest',
            '--prebuilt-image=2.6.9=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.6.9:latest',
+           '--prebuilt-image=2.6.9=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.6.10:latest',
```
Comment (amurciasu) [inline ruby-pipeline/ruby-latest.yaml:65]:
```suggestion
           '--prebuilt-image=2.6.10=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.6.10:latest',
```

File: ruby-pipeline/ruby-latest.yaml [hunk 2]
```diff
@@ -62,17 +62,20 @@ steps:
...
            '--prebuilt-image=2.7.0=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.0:latest',
            '--prebuilt-image=2.7.1=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.1:latest',
            '--prebuilt-image=2.7.2=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.2:latest',
            '--prebuilt-image=2.7.3=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.3:latest',
            '--prebuilt-image=2.7.4=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.4:latest',
            '--prebuilt-image=2.7.5=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.5:latest',
+           '--prebuilt-image=2.7.5=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.6:latest',
```
Comment (amurciasu) [inline ruby-pipeline/ruby-latest.yaml:72]:
```suggestion
           '--prebuilt-image=2.7.6=gcr.io/gcp-runtimes/ruby/ubuntu16/prebuilt/ruby-2.7.6:latest',
```

**Review by amurciasu — CHANGES_REQUESTED** [review]

**Review by amurciasu — APPROVED** [review]

**Comment by ain** [comment]
@dazuma are you perhaps available to merge this one as well? 

`2.7.6` included [CVE-2022-28739](https://www.ruby-lang.org/en/news/2022/04/12/buffer-overrun-in-string-to-float-cve-2022-28739/) that would make it quite relevant to ship, replacing the former pre-built images.

**Comment by dazuma** [comment]
This was handled by #222. The runtime release will happen next week, either June 7 or June 8.

**Pull request closed without merging**